packages/bonito-core/src/__tests__/setup-tests.ts

```diff
@@ -1,15 +1,4 @@
 import { destroyEnvironment } from "../environment";
```

Comment:
I'm all for factoring out our JSDOM stubs/mocks, but at least some of that needs to be in bonito-core since HttpClient needs both fetch() and Blob.

Reply:
Yeah, that's the problem, do you think it's OK to move `jsdom-mocks.ts` into bonito-core or just the blob and fetch?

Reply:
I'm a little torn, but if we want to keep saying that bonito-core can work in Node.js we should probably split them. Node does have Blob and fetch, but won't likely ever implement `matchMedia` or `HTMLCanvasElement`.